### ISSUE TITLE
Can't get GPO

### ISSUE THREAD

**Issue by cabiamdos** [cuamckuu/nfc-frog#13]
Hello,
so basically I have a VISA card (from bbva es) that says "Can't get AFL" when executing "sudo ./nfc-frog GPO".
It has happened with other software readers (like the proxmark reader) and one android reader, but other android reader could successfully read the card.
Any ideas?
Thanks

**Comment by cuamckuu** [comment]
Hi, you could try another modes:
```
sudo ./nfc-frog fast # Fast brute mode
# or
sudo ./nfc-frog full # Full brute mode
```

**Comment by cabiamdos** [comment]
Yes, I have tried all of them, but they end up in errors or the program freeze..
Also, is there a way to run this on windows with mingw or something?
Thanks

**Comment by cuamckuu** [comment]
You could use it, if you could install libnfc on Windows, and connect pn532 there with propper drivers.

Also, check output of `nfc-pool` and `nfc-list` for your card, is it working propperly?

**Comment by cabiamdos** [comment]
I don't have pn532,  I have the acr122u (I believe it is a cloned version because I got it for cheap, but it works)
I installed your program on a raspberry pi, I tried on windows with mingw (bash) but I wasn't able to make it work.
But it just happen with one bbva card I have, and it happens with some readers and other one it works fine. Others card I have works awesome. I will be following your steps and tell you

**Comment by cabiamdos** [comment]
Well, I haven't been able to run it on windows.
I don't understand what output of `nfc-pool` or `nfc-list` you refer to. 

Here is the issue with this VISA card:
https://www.mediafire.com/view/p4s8xaevcpkhfrb/photo_2021-12-25_00-36-38.jpg/file

It has happened the same issue with others MASTERCARD I have, but this one isn't activated or doesn't have any previous transaction.

Any ideas?

**Comment by cuamckuu** [comment]
After installing `libnfc` (which is required to use nfc-frog), you will also get `nfc-pool` and `nfc-list` installed, so check if they work propperly and can read card info.

Also check if card is close enough to reader and that it don't move while reading

**Comment by cabiamdos** [comment]
For windows, to install libnfc I have to install cmake (https://github.com/nfc-tools/libnfc/blob/master/README-Windows.md). It is looking long to me. I still don't know how to use nfc-pool and nfc-list on my raspberry pi (linux)

The card reader and the card doesn't move. 

I have read that MSD transactions are getting obsolete for VISA when trying to emulate the POS terminal to read the data, I have; that could e the reason. But this just happens with one card.
Maybe it can be implemented more options like this:
 Transaction type - qVSDC or M/Chip.
 Transaction type - qVSDC or M/Chip plus CDA (SDAD generation).
 Transaction type - VSDC. For test only. Not a standard behavior.
 VISA. generate AC from GPO.

**Comment by cuamckuu** [comment]
```
root@kali:~# nfc-list -h
nfc-list uses libnfc 1.8.0
usage: nfc-list [-v] [-t X]
```

```
root@kali:~# nfc-poll -h
nfc-poll uses libnfc 1.8.0
usage: nfc-poll [-v]
  -v	 verbose display
```
